### DISCUSSION TITLE
Can't activate Built-in Magnifier Widget by editing view.xml [Parent Theme: Breeze Evolution Theme]

### DISCUSSION THREAD

**Discussion by ghost** [breezefront/community#3] [1. Help]
Hello Support Team,

I am using my own created theme based on Breeze Evolution Theme version 1.1.2 as parent theme on Magento 2.4.4 Open Source, I am not able to activate Built-in Magnifier Widget (Product Image Zoom) by editing view.xml . Can you please let me know the guideline to solve this issue?

Thank you very much!!

Wish you all the best!

**Comment by vovayatsyuk** [comment]
Sorry, but the magnifier is not supported by the breeze gallery.

**Comment by vovayatsyuk** [comment]
We've just released a new Breeze version (2.1.1) that supports magnifier! 

Please update module/theme and enable it from view.xml: https://github.com/breezefront/theme-frontend-breeze-evolution/blob/master/etc/view.xml#L167-L169
All available options: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/etc/view.xml#L249-L266
